shell.nix

```diff
@@ -0,0 +1,8 @@
+{ pkgs ? import (fetchTarball "https://github.com/NixOS/nixpkgs/archive/eabc38219184cc3e04a974fe31857d8e0eac098d.tar.gz") {} }:
```

Comment:
I didn't want to download homebrew just to add the heroku cli. so I added a .nix file. This is currently just for my env. Can add to .gitignore but figured I'd include in case we all use nix one day

if you run 'nix-shell' in the project directory it'll open a shell with heroku cli and postgres available

Reply:
Maybe we get this note checked in somewhere? I think as long as we aren't required to use nix I'm cool if it's convenient for you to check it in